frontend/js/src/user/components/PinnedRecordingCard.tsx

```diff
@@ -148,6 +151,22 @@ export default class PinnedRecordingCard extends React.Component<
           action={() => this.unpinRecording()}
         />
       );
+      additionalMenuItems.push(
+        <ListenControl
+          text="Edit Comment"
+          key="Edit Comment"
+          icon={faPencilAlt}
+          action={() => {
+            NiceModal.show(PinRecordingModal, {
+              recordingToPin: listen,
+              isUpdate: true,
```

Comment:
I wonder if `isUpdate` is actually required here.
Wouldn't the presence of the `rowId` be enough to determine it?
Something like this in PinRecordingModal.tsx:
`const isUpdate = Boolean(rowId)`